default/species.txt

```diff
@@ -1,4 +1,4 @@
-include "shared_macros.txt"
```

Comment:
wasn't it preferred to put the includes at the end of the file to preserve the line numbering for parse error output?